core/embed/lib/gfx_color.h

```diff
@@ -342,4 +342,9 @@ const gfx_color16_t* gfx_color16_gradient_a4(gfx_color_t fg, gfx_color_t bg);
 // the background (`bg`) color and `retval[15]` the foreground (`fg`) color
 const gfx_color32_t* gfx_color32_gradient_a4(gfx_color_t fg, gfx_color_t bg);
 
+// Returns a color with alpha channel set
+//
+// The original color is not modified
+gfx_color32_t gfx_color32_set_alpha(gfx_color32_t c, uint8_t alpha);
```

Comment:
If we improved `gfx_color32_blend_a4/a8()` so that they could interpolate the alpha channel as well, we could eliminate this function altogether.

Reply:
the `gfx_color32_blend_a4/a8` with `GFX_COLOR_16BIT` setting do not have alpha channels in the input params, so this is kinda impractical

we would need to use the 32bit variant, convert the colors from 16 to 32bit format and still set alpha on one of the values

Reply:
or add `start_alpha` `end_alpha` parameters to 16bit variant of `gfx_color32_blend_a4`, which also isnt very nice

Reply:
Yes, you are right, it wasn't good idea at all. Maybe, then if we need `gfx_color32_set_alpha()`, we can make it `static inline`

Reply:
done here https://github.com/trezor/trezor-firmware/commit/2f741408ba9e087a6dc87a94c53c81831e7837d5